spec/views/ops/_zone_form.html.haml_spec.rb

```diff
@@ -2,7 +2,7 @@
   before do
     assign(:sb, :active_tab => "settings_evm_servers")
     @selected_zone = FactoryBot.create(:zone, :name => 'One Zone', :description => " One Description", :settings =>
-                                                {:proxy_server_ip => '1.2.3.4', :concurrent_vm_scans => 0, :ntp => {:server => ['Server 1']}})
```

Comment:
failure is over here


rspec ./spec/views/ops/_zone_form.html.haml_spec.rb:30

```
expect(response.body).to include('NTP Servers')
```